packages/ui/src/components/AccountEditName.tsx

```diff
@@ -1,77 +1,68 @@
-import { Grid, TextField } from "@mui/material"
-import { ChangeEvent, useCallback, useEffect, useMemo, useState } from "react"
-import { styled }  from "@mui/material/styles"
-import { useAccounts } from "../contexts/AccountsContext"
-import { useAccountNames } from "../contexts/AccountNamesContext"
-import AccountDisplay from "./AccountDisplay"
+import { Grid, TextField } from '@mui/material';
+import { ChangeEvent, useCallback, useEffect, useMemo, useState } from 'react';
+import { styled } from '@mui/material/styles';
+import { useAccounts } from '../contexts/AccountsContext';
+import { useAccountNames } from '../contexts/AccountNamesContext';
+import AccountDisplay from './AccountDisplay';
 
 export interface OnChangeArgs {
-    addresses: (string | undefined)[]
-    name: string
+  addresses: (string | undefined)[];
+  name: string;
 }
 
 interface Props {
-    className?: string
-    address: string
-    onNameChange: (args: OnChangeArgs) => void
+  className?: string;
+  address: string;
+  onNameChange: (args: OnChangeArgs) => void;
 }
 
 const AccountEditName = ({ address, onNameChange, className }: Props) => {
-  const { getNamesWithExtension } = useAccountNames()
-  const { addressList } = useAccounts()
-  const [name, setName] = useState("")
-  const isExtensionAccount = useMemo(() => addressList.includes(address), [address, addressList])
+  const { getNamesWithExtension } = useAccountNames();
+  const { addressList } = useAccounts();
+  const [name, setName] = useState('');
+  const isExtensionAccount = useMemo(
+    () => addressList.includes(address),
+    [address, addressList]
+  );
 
   useEffect(() => {
     if (!name) {
-      const defaultName = getNamesWithExtension(address) || ""
-      setName(defaultName)
+      const defaultName = getNamesWithExtension(address) || '';
+      setName(defaultName);
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [getNamesWithExtension])
+  }, [getNamesWithExtension]);
 
-  const onChange = useCallback((event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
-    const value = event.target.value
-    setName(value)
-    onNameChange({ addresses: [address], name: value })
-  }, [address, onNameChange])
+  const onChange = useCallback(
+    (event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+      const value = event.target.value;
```

Comment:
personal opinion, I find `;` useless. Can we add a rule to enforce no semi-colon please?